src/IdealizedExperiments/eddying_channel.jl

```diff
@@ -218,9 +218,8 @@ function eddying_channel_simulation(;
     simulation = Simulation(model; Δt=initial_Δt, stop_time)
 
     # add timestep wizard callback
-    wizard = TimeStepWizard(; cfl = 0.1, max_change = 1.01, max_Δt)
+    wizard = TimeStepWizard(; cfl = 0.2, max_change = 1.01, max_Δt)
```

Comment:
`max_change = 1.01`?
Is this required?

Reply:
It's helpful when experimenting with _very_ small initial time-steps for CATKE